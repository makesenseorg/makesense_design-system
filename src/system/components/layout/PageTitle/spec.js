import { shallowMount } from '@vue/test-utils'
import Comp from './PageTitle.vue'

describe('PageTitle.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Comp, {
      propsData: {
        heading: ''
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})