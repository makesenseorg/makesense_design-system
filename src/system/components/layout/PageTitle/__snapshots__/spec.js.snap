// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle.vue matches snapshot 1`] = `
<header
  class="ds-page-title"
>
  <ds-container-stub>
    <ds-heading-stub>
      
      
    
    </ds-heading-stub>
     
  </ds-container-stub>
</header>
`;
